test/tests/parallel/set1/spam_virus/spam_junk_folder.bats

```diff
@@ -69,6 +72,28 @@ function teardown() { _default_teardown ; }
   _should_receive_spam_at '/var/mail/localhost.localdomain/user1/.Junk/new/'
 }
 
+@test "(enabled + MARK_SPAM_AS_READ=1) should mark spam message as read" {
+  export CONTAINER_NAME=${CONTAINER2_NAME}
+
+  local CUSTOM_SETUP_ARGUMENTS=(
+    --env ENABLE_AMAVIS=1
+    --env ENABLE_SPAMASSASSIN=1
+    --env SA_SPAM_SUBJECT="SPAM: "
+    --env SPAMASSASSIN_SPAM_TO_INBOX=1
+    --env MOVE_SPAM_TO_JUNK=0
+    --env MARK_SPAM_AS_READ=1
+    --env PERMIT_DOCKER=container
+  )
+  _init_with_defaults
+  _common_container_setup 'CUSTOM_SETUP_ARGUMENTS'
+
+  _should_send_spam_message
+  _should_be_received_by_amavis 'Passed SPAM {RelayedTaggedInbound,Quarantined}'
+
+  # Should move delivered spam to INBOX as read (cur instead of new)
+  _should_receive_spam_at '/var/mail/localhost.localdomain/user1/cur/'
```

Comment:
Now the default `MOVE_SPAM_TO_JUNK=1` should apply, and this should have moved the spam mail into the users junk inbox as "read":

```suggestion
  # Should move delivered spam to Junk folder as read (`cur` instead of `new`)
  _should_receive_spam_at '/var/mail/localhost.localdomain/user1/Junk/cur/'
```